buildbuddy.yaml

```diff
@@ -62,6 +62,22 @@ actions:
       - "run --config=workflows //test/fixtures:validate"
       - "build --config=workflows //..."
 
+  - name: Integration Test - "examples/sanitizers"
```

Comment:
Can you move this above `"examples/simple"` to keep them alphabetical?

Reply:
Oh yes :) Updated!